### DISCUSSION TITLE
How can I take a log of the probability values in the Conditional probability table?

### DISCUSSION THREAD

**Discussion by kenneth-lee-ch** [agrumery/aGrUM#4] [Q&A] [answered]
I tried to use the numpy package, but it throws out an error because the data type of the cpt is potential. I tried to convert the table to the array, but it also doesn't seem to work either. Any help will be appreciated!

**Comment by phwuil** [comment, marked as answer]
Hello , 

I do not understand your problem :

![image](https://user-images.githubusercontent.com/5980636/215987929-06a2efb5-fcc2-4b2f-ac6e-0365cda08d74.png)

?

**Comment by phwuil** [comment]
And finally : 

![image](https://user-images.githubusercontent.com/5980636/215990079-89523263-9549-4d25-a725-e651b3b18f94.png)

**Comment by kenneth-lee-ch** [comment]
Oh, I don't know that you can do p.log2(). Thanks!

**Comment by phwuil** [comment]
https://pyagrum.readthedocs.io/en/1.5.2/potential.html#pyAgrum.Potential.log2
;-)
